packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts

```diff
@@ -88,6 +88,14 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
    * in case an option does not come with user-defined custom styles
    */
   comboBoxOptionStyles?: Partial<IComboBoxOptionStyles>;
+
+  /**
+   * Callout width is based on parent width
+   * When set to false the callout will be sized based on the width of the callout's content
+   * @default true;
+   */
+  calloutFixedWidth?: boolean;

```

Comment:
I think this is something we'd want as an option on the comboBoxOptionStyles instead of being passed in explicitly as a prop.